src/index.ts

```diff
@@ -1,13 +1,11 @@
-import * as extend from 'extend';
-import * as _request from 'request';  // for types only
-import * as request from 'retry-request';
```

Comment:
We seem to have lost the automatic retry functionality here.

Reply:
Funny story!  We didn't.  The retry lib was used, but it appears to be configured to do 0 retries:
https://github.com/stephenplusplus/gcp-metadata/blob/master/src/index.ts#L39

@stephenplusplus  am I understanding this wrong?

Reply:
You're misunderstanding. We do retry on http errors. We avoid retrying on network errors (e.g. DNS resolution failure).